### ISSUE TITLE
Dev env broken after stylint 14 support

### ISSUE THREAD

**Issue by devTeaa** [AndyOGo/stylelint-declaration-strict-value#249]
Hi, I would like to contribute on this project, but test are failing with other ts errors, I managed to track it down to when this happens and it was this commit point:

37904375f9d508f3f6c15eac39caafd8b9d991c5

before that test are working fine, is there a chance this get fixed?

**Comment by AndyOGo** [comment]
Thanks for your issue @devTeaa 

May I ask you to be more specific?
Please provide error logs, OS, node, NPM versions, etc.

All is OK on my local machine.
I'm on macOS cataline version `10.15.7`.

```sh
$ node -v
v15.10.0

$ npm -v
7.5.3

$ npm test

> stylelint-declaration-strict-value@1.8.0 test
> node --require babel-register-ts test

TAP version 13

...

1..777
# tests 777
# pass  777

# ok
```

**Comment by devTeaa** [comment]
Hi @AndyOGo , I have matched my node and npm version with yours and this is my log


```bash
not ok 387 test exited without ending: > rule: scale-unlimited/declaration-strict-value > config: "/color/", {"recurseLonghand":"foo"} > code: ".foo { border: red; }"
  ---
    operator: fail
    at: process.<anonymous> (/mnt/d/_windows/stylelint-declaration-strict-value/node_modules/tape/index.js:86:23)
    stack: |-
      Error: test exited without ending:
      > rule: scale-unlimited/declaration-strict-value
      > config: "/color/", {"recurseLonghand":"foo"}
      > code: ".foo { border: red; }"

          at Test.assert [as _assert] (/mnt/d/_windows/stylelint-declaration-strict-value/node_modules/tape/lib/test.js:275:54)
          at Test.bound [as _assert] (/mnt/d/_windows/stylelint-declaration-strict-value/node_modules/tape/lib/test.js:89:32)
          at Test.fail (/mnt/d/_windows/stylelint-declaration-strict-value/node_modules/tape/lib/test.js:368:10)
          at Test.bound [as fail] (/mnt/d/_windows/stylelint-declaration-strict-value/node_modules/tape/lib/test.js:89:32)
          at Test._exit (/mnt/d/_windows/stylelint-declaration-strict-value/node_modules/tape/lib/test.js:238:14)
          at Test.bound (/mnt/d/_windows/stylelint-declaration-strict-value/node_modules/tape/lib/test.js:89:32)
          at process.<anonymous> (/mnt/d/_windows/stylelint-declaration-strict-value/node_modules/tape/index.js:86:23)
          at process.emit (events.js:215:7)
  ...

1..387
# tests 387
# pass  24
# fail  363
```

**Comment by AndyOGo** [comment]
Hi @devTeaa 

Thanks for your quick response.

Wau, I assume you executed `npm test` and get only `387` tests instead of `777` where almost all are failing 🤔 

Honestly I never had such issue son macOS...
To me it looks like to be either an OS issue, WSL or `tape`...
- https://github.com/substack/tape/issues/223
- https://github.com/substack/tape/issues/160
